lib/cocoapods-binary-cache/pod-rome/xcodebuild_command.rb

```diff
@@ -19,8 +19,18 @@ def initialize(options)
     end
 
     def run
-      sdks.each { |sdk| build_for_sdk(sdk) }
+      if !should_archive?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/grab/cocoapods-binary-cache/blob/b144f30fb41bd34e49fceb072710e5f2c3a3e710/lib/cocoapods-binary-cache/pod-rome/xcodebuild_command.rb#L22"></span>Style/NegatedIf: Favor <code>unless</code> over <code>if</code> for negative conditions.</td>
    </tr>
  </tbody>
</table>